src/components/action-group/styles.scss

```diff
@@ -1,25 +1,14 @@
-
-// ==========================================================================
-// Vlaanderen UI -- ActionGroup
-// ==========================================================================
 @charset "utf-8";
-
-// ==========================================================================
-// Core
-// ==========================================================================
 @import "../../../node_modules/@govflanders/vl-ui-core/src/scss/core";
-
-// ==========================================================================
-// Components
-// ==========================================================================
 @import "../../../node_modules/@govflanders/vl-ui-action-group/src/scss/action-group";
-
-// ==========================================================================
-// Util
-// ==========================================================================
 @import "../../../node_modules/@govflanders/vl-ui-util/src/scss/util";
 
-.vl-action-group > *,
 .vl-action-group > ::slotted(*) {
-  @extend .vl-button
+  margin-right: 1.4rem;
```

Comment:
Misschien voor deze extra styling stukken geen slecht idee om er commentaar bij te zetten dat zegt waarom het er gezet is